backend/rent.py

```
@@ -0,0 +1,158 @@
+                        
+                    else:
+                        flash('Login Successfull')
+                        return redirect(url_for('login'))
+                else:
+                    message = 'Incorrect Password'
+                    if request.is_json:
+                        return jsonify({"message": message, "status": "error"}), 401
+            else:
+                flash("Login Unsuccessful")
+                return redirect(url_for('login'))
+        except mysql.connector.Error as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)